crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -465,4 +473,11 @@ mod tests {
             "../node_modules/@sentry/browser/esm/helpers.js"
         );
     }
+
+    #[test]
+    fn test_join_paths() {
+        let base = "http://example.com".parse().unwrap();
+        let path = "webpack:///../node_modules/scheduler/cjs/scheduler.production.min.js";
```

Comment:
an explicit test for `/./` would be nice as well :-) the more tests the better :-)